src/FSharpVSPowerTools.Logic/QuickInfoMargin.fs

```diff
@@ -136,7 +136,7 @@ type QuickInfoMargin (textDocument: ITextDocument,
             |> Async.StartInThreadPoolSafe
         | None -> updateQuickInfo (None, None)
 
-    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 1000us, updateAtCaretPosition)
+    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 200us, updateAtCaretPosition)
```

Comment:
Shouldn't https://github.com/fsharp/FSharp.Compiler.Service/pull/445 fix it so we can now don't worry about number of requests? 

Many of our features use 200..500 ms delay.